### PR TITLE
list .venv when it exists

### LINKED ISSUE #1759: unexpected empty environments listing
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [ ] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option).

- **OS version and name**: macos 10.15.2
- **Poetry version**: 1.0.0

## Issue

Surprisingly `poetry env list` does not return anything. Virtualenv was created by `poetry install` in a local directory using local `poetry.toml` 

```
[virtualenvs]
in-project = true
```

Output:
```
 ~/Projects/fooproject ❯ poetry env use 3.7.4                                                                                                                                                     3.7.4
Using virtualenv: /Users/Silver/Projects/fooproject/.venv
 ~/Projects/fooproject  ❯ poetry env list                                                                                                                                                          3.7.4
 ~/Projects/fooproject  ❯ poetry env list --full-path                                                                                                                                              3.7.4
 ~/Projects/fooproject  ❯ poetry env info                                                                                                                                                          3.7.4

Virtualenv
Python:         3.7.4
Implementation: CPython
Path:           /Users/Silver/Projects/fooproject/.venv
Valid:          True

System
Platform: darwin
OS:       posix
Python:   /Users/Silver/.pyenv/versions/3.7.4
 ~/Projects/fooproject ❯ poetry --version                                                                                                                                                         3.7.4
Poetry version 1.0.0
```

### PR DESCRIPTION
**Pull request by frostming**
# Pull Request Check List

This is just a reminder about the most common mistakes. Please make sure that you tick all *appropriate* boxes.  But please read our [contribution guide](https://python-poetry.org/docs/contributing/) at least once, it will save you unnecessary review cycles!

Fix #1759 

- [x] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

**Note**: If your Pull Request introduces a new feature or changes the current behavior, it should be based
on the `develop` branch. If it's a bug fix or only a documentation update, it should be based on the `master` branch.

If you have *any* questions to *any* of the points above, just **submit and ask**!  This checklist is here to *help* you, not to deter you from contributing!

### CODE REVIEW

**Comment by frostming** [comment]
Question: How do I keep the in-dev changelog?

File: poetry/console/commands/env/list.py [hunk 1]
```diff
@@ -15,8 +15,14 @@ def handle(self):
...
         current_env = manager.get()
-
-        for venv in manager.list():
+        env_list = manager.list()
+        if (
+            self.poetry.config.get("virtualenvs.in-project")
+            and current_env not in env_list
+        ):
+            env_list.insert(0, current_env)
+
```
Comment (finswimmer) [inline poetry/console/commands/env/list.py]: Thanks a lot for your contribution. :+1: 

I wonder if this check shouldn't be implemented directly in the `list()` method.

Comment (frostming) [reply poetry/console/commands/env/list.py]: Makes sense, PR updated.

File: poetry/utils/env.py [hunk 2]
```diff
@@ -367,11 +367,19 @@ def list(self, name=None):  # type: (Optional[str]) -> List[VirtualEnv]
...
-        return [
+        env_list = [
             VirtualEnv(Path(p))
             for p in sorted(venv_path.glob("{}-py*".format(venv_name)))
         ]
 
+        current_env = self.get()
+        if (
+            self._poetry.config.get("virtualenvs.in-project")
+            and current_env not in env_list
```
Comment (finswimmer) [inline poetry/utils/env.py]: Thanks a lot again :+1: 

One more thing: The current check adds the the current activated env to the list. But what we want, is to add `.venv` if it exists in the project folder, isn't it? The current path to the project folder should be available with `self._poetry.file.path.parent` at this point.

**Comment by sdispater** [comment]
Thanks!

**Pull request merged**